### PR TITLE
Update deploy-jobs.md

### PR DESCRIPTION
**Pull request by Tonayya**
## What are you changing in this pull request and why?

Added some examples where we are using the L expression to specify the last day of the month. Relevant conversation can be found in Slack [here](https://dbt-labs.slack.com/archives/C04KKUHLT32/p1702002328874979).
## Checklist

- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.

### CODE REVIEW

**Comment by Tonayya** [comment]
Hallo @mirnawong1 please let me know if this all good and if I need to change anything. Thank you love your work <3

File: website/docs/docs/deploy/deploy-jobs.md [hunk 1]
```diff
@@ -87,12 +87,19 @@ Use tools such as [crontab.guru](https://crontab.guru/) to generate the correct
...
 
-- `0 * * * *`: Every hour, at minute 0
-- `*/5 * * * *`: Every 5 minutes
-- `5 4 * * *`: At exactly 4:05 AM UTC
-- `30 */4 * * *`: At minute 30 past every 4th hour (e.g. 4:30AM, 8:30AM, 12:30PM, etc., all UTC)
-- `0 0 */2 * *`: At midnight UTC every other day
+- `0 * * * *`: Every hour, at minute 0.
+- `*/5 * * * *`: Every 5 minutes.
+- `5 4 * * *`: At exactly 4:05 AM UTC.
+- `30 */4 * * *`: At minute 30 past every 4th hour (e.g. 4:30AM, 8:30AM, 12:30PM, etc., all UTC).
```
Comment (mirnawong1) [inline website/docs/docs/deploy/deploy-jobs.md]:
```suggestion
- `30 */4 * * *`: At minute 30 past every 4th hour (such as 4:30 AM, 8:30 AM, 12:30 PM, and so on, all UTC).
```

File: website/docs/docs/deploy/deploy-jobs.md [hunk 2]
```diff
@@ -87,12 +87,19 @@ Use tools such as [crontab.guru](https://crontab.guru/) to generate the correct
...
-- `5 4 * * *`: At exactly 4:05 AM UTC
-- `30 */4 * * *`: At minute 30 past every 4th hour (e.g. 4:30AM, 8:30AM, 12:30PM, etc., all UTC)
-- `0 0 */2 * *`: At midnight UTC every other day
+- `0 * * * *`: Every hour, at minute 0.
+- `*/5 * * * *`: Every 5 minutes.
+- `5 4 * * *`: At exactly 4:05 AM UTC.
+- `30 */4 * * *`: At minute 30 past every 4th hour (such as 4:30 AM, 8:30 AM, 12:30 PM, and so on, all UTC).
+- `0 0 */2 * *`: At midnight UTC every other day.
 - `0 0 * * 1`: At midnight UTC every Monday.
+- `0 0 L * *`: At 12:00 AM, on the last day of the month.
```
Comment (mirnawong1) [inline website/docs/docs/deploy/deploy-jobs.md]:
```suggestion
- `0 0 L * *`: At 12:00 AM (midnight), on the last day of the month.
```

File: website/docs/docs/deploy/deploy-jobs.md [hunk 3]
```diff
@@ -87,12 +87,19 @@ Use tools such as [crontab.guru](https://crontab.guru/) to generate the correct
...
-- `0 * * * *`: Every hour, at minute 0
-- `*/5 * * * *`: Every 5 minutes
-- `5 4 * * *`: At exactly 4:05 AM UTC
-- `30 */4 * * *`: At minute 30 past every 4th hour (e.g. 4:30AM, 8:30AM, 12:30PM, etc., all UTC)
-- `0 0 */2 * *`: At midnight UTC every other day
+- `0 * * * *`: Every hour, at minute 0.
+- `*/5 * * * *`: Every 5 minutes.
+- `5 4 * * *`: At exactly 4:05 AM UTC.
+- `30 */4 * * *`: At minute 30 past every 4th hour (such as 4:30 AM, 8:30 AM, 12:30 PM, and so on, all UTC).
+- `0 0 */2 * *`: At midnight UTC every other day.
```
Comment (mirnawong1) [inline website/docs/docs/deploy/deploy-jobs.md]:
```suggestion
- `0 0 */2 * *`: At 12:00 AM (midnight) UTC every other day.
```

File: website/docs/docs/deploy/deploy-jobs.md [hunk 4]
```diff
@@ -87,12 +87,19 @@ Use tools such as [crontab.guru](https://crontab.guru/) to generate the correct
...
+- `*/5 * * * *`: Every 5 minutes.
+- `5 4 * * *`: At exactly 4:05 AM UTC.
+- `30 */4 * * *`: At minute 30 past every 4th hour (such as 4:30 AM, 8:30 AM, 12:30 PM, and so on, all UTC).
+- `0 0 */2 * *`: At 12:00 AM (midnight) UTC every other day.
 - `0 0 * * 1`: At midnight UTC every Monday.
+- `0 0 L * *`: At 12:00 AM (midnight), on the last day of the month.
+- `0 0 L 1,2,3,4,5,6,8,9,10,11,12 *`: At 12:00 AM, on the last day of the month, only in January, February, March, April, May, June, August, September, October, November, and December.
+- `0 0 L 7 *`: At 12:00 AM, on the last day of the month, only in July.
+- `0 0 L * FRI,SAT`: At 12:00 AM, on the last day of the month, and on Friday and Saturday.
+- `0 12 L * *`: At 12:00 PM, on the last day of the month.
```
Comment (mirnawong1) [inline website/docs/docs/deploy/deploy-jobs.md]:
```suggestion
- `0 12 L * *`: At 12:00 PM (afternoon), on the last day of the month.
```

**Review by mirnawong1 — APPROVED** [review]
hey @Tonayya nice one and thank you for opening this pr up to clarify/add examples. super useful and thank you for contributing. going to merge this fo ryou!

File: website/docs/docs/deploy/deploy-jobs.md [hunk 5]
```diff
@@ -87,12 +87,19 @@ Use tools such as [crontab.guru](https://crontab.guru/) to generate the correct
 Refer to the following example snippets:
```
Comment (mirnawong1) [inline website/docs/docs/deploy/deploy-jobs.md]:
```suggestion
Here are examples of cron job schedules. The dbt Cloud job scheduler supports using `L` to schedule jobs on the last day of the month:
```

**Comment by Tonayya** [comment]
Hi @mirnawong1 sorry I closed the PR because I thought changes were already deployed. But it doesn't look like it was in the docs. Did I do something wrong? :(

**Comment by mirnawong1** [comment]
hey @Tonayya  all good! i think i was going to merge it after the slack approval/clarification wrt emea/au MT. i'll merge this for you and will save you that step 🙏

**Pull request merged**